deploy/README.md

```diff
@@ -197,56 +343,44 @@ kubectl -n sumologic apply -f fluentd-sumologic.yaml
 The manifest will create the Kubernetes resources required by Fluentd.
 
 
-### Verify the pods are running
+#### Verify the pods are running
 
 ```sh
 kubectl -n sumologic get pod
 ```
 
-## Step 2: Configure Prometheus
+### Step 2: Configure Prometheus
 
 In this step, you will configure the Prometheus server to write metrics to Fluentd.
 
 Install Helm:
 
 *Note the following steps are one way to install Helm, but in order to ensure property security, please be sure to review the [Helm documentation.](https://helm.sh/docs/using_helm/#securing-your-helm-installation)*
 
-```sh
-brew install kubernetes-helm
```

Comment:
Let's keep this step, since the users will need to install helm.

But let's also include a note here saying that we're only downloading helm to generate the yaml files necessary to deploy, NOT to actually deploy anything.